.github/workflows/docker.yml

```
@@ -9,24 +9,29 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
       #
       - name: Set up QEMU
-        uses: docker/setup-qemu-action@v1
+        uses: docker/setup-qemu-action@v2
       #
       - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v1
+        uses: docker/setup-buildx-action@v2
       #
       - name: Login to DockerHub
+        env:
+          MY_SECRET: ${{ secrets.DOCKERHUB_USERNAME }}
+        if: env.MY_SECRET != null
         uses: docker/login-action@v1
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
       #
       - name: Build and push
+        env:
+          MY_SECRET: ${{ secrets.DOCKERHUB_USERNAME }}
         uses: docker/build-push-action@v2
         with:
           file: Dockerfile
           context: .
-          push: true
-          tags: mdouchement/openstackswift:latest
+          push: ${{ fromJSON(env.MY_SECRET != null) }}
+          tags: ${{ github.repository }}:latest
```

How about building for both Debian & Alpine?

```suggestion
          tags: ${{ github.repository }}:latest
          
      - name: Build and push (Alpine)
        env:
          DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
        uses: docker/build-push-action@v4
        with:
          file: Dockerfile.alpine
          context: .
          push: ${{ fromJSON(env.DOCKERHUB_USERNAME != null) }}
          tags: ${{ github.repository }}:alpine
```